backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/job/service/commands/PickingJobCreateCommand.java

```diff
@@ -189,6 +191,8 @@ private static PackageableQuery toPackageableQuery(@NonNull final PickingJobCrea
 				.deliveryBPLocationId(request.getDeliveryBPLocationId())
 				.warehouseTypeId(request.getWarehouseTypeId())
 				.onlyShipmentScheduleIds(request.getShipmentScheduleIds())
+				.maximumFixedPreparationDate(getZonedDateTime())
+				.maximumFixedPreparationDate(getZonedDateTime())
```

Comment:
extract as field

Reply:
extracting as local variable, because method is static.